tests/integration/features/istio/production/main-suite/istio_compatibility_mode.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Compatibility Mode
+
+  Background:
+    Given "Istio CR" is not present on cluster
+    And Evaluated cluster size is "Production"
+    And Istio CRD is installed
+    And Namespace "istio-system" is "not present"
+    And "Deployment" "istio-controller-manager" in namespace "kyma-system" is ready
+
+  Scenario: Compatibility mode is set on
+    Given Istio CR "istio-sample" from "istio_cr_template" is applied in namespace "kyma-system"
+    And Istio CR "istio-sample" in namespace "kyma-system" has status "Ready"
+    # ENABLE_EXTERNAL_NAME_ALIAS is set to true by the custom annotation handler, because of that we need to verify if it's not overwritten
```

Comment:
This is confusing as we are verifying here that it's true and stating we need to verify that it's not overwritten and later we check if it has value false.

Reply:
It's the only compatibility variable affected by the custom annotation impl. We could also drop this step, but I decided to include it so we have a full coverage of the behaviour. Also I can improve the comment, to more precisely describe what's going on. What's your proposal?

Reply:
You should keep the step, but the comment is confusing for me.

Reply:
Then I'll improve it